### PR TITLE
Deprecate getLastLogin, replace with java 8 getLastLoginTime

### PR DESCRIPTION
**Pull request by omnivion**

### CODE REVIEW

File: src/main/java/fr/xephi/authme/api/v3/AuthMeApi.java [hunk 1]
```diff
@@ -164,13 +166,24 @@ public String getLastIp(String playerName) {
      * @param playerName The name of the player to process
      * @return The date of the last login, or null if the player doesn't exist or has never logged in
      */
+    @Deprecated
```
Comment (games647) [inline src/main/java/fr/xephi/authme/api/v3/AuthMeApi.java]: Please document the alternative method in JavaDoc deprecated tag.

File: src/main/java/fr/xephi/authme/api/v3/AuthMeApi.java [hunk 2]
```diff
@@ -164,13 +166,24 @@ public String getLastIp(String playerName) {
...
     public Date getLastLogin(String playerName) {
-        PlayerAuth auth = playerCache.getAuth(playerName);
+        Long lastLogin = getLastLoginMillis(playerName);
+        return lastLogin == null ? null : new Date(lastLogin);
+    }
+
+    public Instant getLastLoginTime(String playerName) {
```
Comment (games647) [inline src/main/java/fr/xephi/authme/api/v3/AuthMeApi.java]: API methods should have JavaDoc comments.

File: src/main/java/fr/xephi/authme/api/v3/AuthMeApi.java [hunk 3]
```diff
@@ -164,13 +166,24 @@ public String getLastIp(String playerName) {
...
+        Long lastLogin = getLastLoginMillis(playerName);
+        return lastLogin == null ? null : Instant.ofEpochMilli(lastLogin);
+    }
+
+    private Long getLastLoginMillis(String playerName) {
+    	    PlayerAuth auth = playerCache.getAuth(playerName);
```
Comment (games647) [inline src/main/java/fr/xephi/authme/api/v3/AuthMeApi.java]: Mixing tab and spaces.

**Comment by games647** [comment]
Great idea

**Comment by sgdc3** [comment]
@omnivion Any news?

**Comment by omnivion** [comment]
@sgdc3 Just got back from vacation. I'll look today.

**Comment by sgdc3** [comment]
@omnivion Any news?

**Comment by ljacqu** [comment]
Thanks for finalizing it, @games647  :)

**Pull request closed without merging**